src/averageAges.js

```diff
@@ -20,6 +20,37 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const theSamecentury = people
+    .filter((person) => {
+      return Math.ceil(person.died / 100) === century
+      && person.sex === 'm';
+    });
```

Comment:
```suggestion
    .filter((person) => {
      return century 
      ? Math.ceil(person.died / 100) === century && person.sex === 'm'
      : person.sex === 'm'
    });
```

Reply:
not needed to calculate both values. it's enough to calculate smth directly, use ternary operator to do it

Reply:
fix it for all functions please